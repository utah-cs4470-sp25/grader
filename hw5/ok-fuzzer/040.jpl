write image ( zP ) to "2_096p_7>l\"   
read image "q" to d  
time show if M9 ( array [ q_8 : array [  ] sum [  ] B,U7A_6 : u ] ( ( ( E ) ) ),( N551R ),e_ ( sum [  ] D . d_Y_,d_2,s . bHR,xP,u_d ),nB ) then t9e else sum [  ] Y { o_ { 5563 },j7G_7 { sum [  ] u,( e ) },yA3_F,nW8__9 { f,l,sum [  ] l } }
show e//7_6\%
//5a"22_7U_&+
write image array [ f_V_8p : - if if 646142 then 7927977 else 9713557 then - a else if sum [  ] b then Y else ( M ),O_W : F { if sum [  ] B then array [  ] c else Z - array [  ] 7435 },C95_i : ( array [  ] array [  ] W . h . W5_cG ),YMO : ! ( W08V_ ( m,W ) ),c7_M : [ A142,! 453910,- 016514235,if ( array [  ] R ) then 74522027 else 2973 != Z ] ] ( [ if Q then s . t else array [  ] Q,array [ z : I,H : W ] V . j,array [  ] array [  ] F,( x ) . vu,( A ) ] ) [ 816224 [ M ] == 912200 [ - f ] ] * - ( eI__P6 ) to ")"
fn   X7_(k5V_[m4K,OB__P] : bool,D[K4_n,Q63_] : int,r_[OW6s_,yS1_L] : f_ [ ,, ]):int{   
  
assert - sum [ U_1I : ( z ),B0F_5 : 965,u7iJ_3 : 4 - T,h : 65542 % ( x ) ] Y__h1 { if t then e else j } , "/v_,&^+5_"

let O_ = kR { 8586212278,array [  ] ( z ),sum [ J8a_v : sum [  ] K,M_T : g,wike2 : S,v : array [  ] o ] array [  ] F + y,Aj4s__ ( array [  ] z ),! M } != [ xi_ ( X ),array [  ] O . n ] >= I_4G8_ { sum [  ] k } 
return q_rx9 { fh_US ( 580513705,t . s,0 ),[ t + array [  ] B,sum [ L : K ] ( G ),if I . G then 721894138 else m,[ I ] ],V254__ ( ! J,! c ) }  
//9_087_T8__<z__07{%k
assert sum [ V418_ : if l_8__8 then 5404 else if Z5z ( c ) then if ( L ) then 610579781 else ( D ) else ( 5523 ),j_ : array [ k : T,F__f : sum [  ] E,Zq0 : 16067912,z7 : L ] sum [  ] array [  ] z . y,X : array [  ] a . f [ 220415908 ] . Y97S_n,S2 : sum [ C : mK98,i_4__ : [ R ],D_3 : ! a,b : 66 >= ( S ),y12_91 : ( 8470082478 ) ] [ m,28875,8320638 ] ] T___p ( j___2 { sum [  ] Y } ) , ""
//8_ZtaM431\C_p_O4
return P_ { - c,( if m then h else n ),! k,( S . R ) } . K1 && array [ wZ : k8 { ( h ),( Q ),E,m . o } ] [ ( t ),21355,X,043 ] && R__ ( 39,x . M,N )  
   
}
