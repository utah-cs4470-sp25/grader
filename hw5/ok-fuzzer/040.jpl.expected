(WriteCmd (VarExpr zP) "2_096p_7>l\")
(ReadCmd "q" (VarLValue d))
(TimeCmd (ShowCmd (IfExpr (CallExpr M9 (ArrayLoopExpr q_8 (ArrayLoopExpr (SumLoopExpr (VarExpr B))) U7A_6 (VarExpr u) (VarExpr E)) (VarExpr N551R) (CallExpr e_ (SumLoopExpr (DotExpr (VarExpr D) d_Y_)) (VarExpr d_2) (DotExpr (VarExpr s) bHR) (VarExpr xP) (VarExpr u_d)) (VarExpr nB)) (VarExpr t9e) (SumLoopExpr (StructLiteralExpr Y (StructLiteralExpr o_ (IntExpr 5563)) (StructLiteralExpr j7G_7 (SumLoopExpr (VarExpr u)) (VarExpr e)) (VarExpr yA3_F) (StructLiteralExpr nW8__9 (VarExpr f) (VarExpr l) (SumLoopExpr (VarExpr l))))))))
(ShowCmd (VarExpr e))
(WriteCmd (ArrayLoopExpr f_V_8p (UnopExpr - (IfExpr (IfExpr (IntExpr 646142) (IntExpr 7927977) (IntExpr 9713557)) (UnopExpr - (VarExpr a)) (IfExpr (SumLoopExpr (VarExpr b)) (VarExpr Y) (VarExpr M)))) O_W (StructLiteralExpr F (IfExpr (SumLoopExpr (VarExpr B)) (ArrayLoopExpr (VarExpr c)) (BinopExpr (VarExpr Z) - (ArrayLoopExpr (IntExpr 7435))))) C95_i (ArrayLoopExpr (ArrayLoopExpr (DotExpr (DotExpr (VarExpr W) h) W5_cG))) YMO (UnopExpr ! (CallExpr W08V_ (VarExpr m) (VarExpr W))) c7_M (ArrayLiteralExpr (VarExpr A142) (UnopExpr ! (IntExpr 453910)) (UnopExpr - (IntExpr 16514235)) (IfExpr (ArrayLoopExpr (VarExpr R)) (IntExpr 74522027) (BinopExpr (IntExpr 2973) != (VarExpr Z)))) (BinopExpr (ArrayIndexExpr (ArrayLiteralExpr (IfExpr (VarExpr Q) (DotExpr (VarExpr s) t) (ArrayLoopExpr (VarExpr Q))) (ArrayLoopExpr z (VarExpr I) H (VarExpr W) (DotExpr (VarExpr V) j)) (ArrayLoopExpr (ArrayLoopExpr (VarExpr F))) (DotExpr (VarExpr x) vu) (VarExpr A)) (BinopExpr (ArrayIndexExpr (IntExpr 816224) (VarExpr M)) == (ArrayIndexExpr (IntExpr 912200) (UnopExpr - (VarExpr f))))) * (UnopExpr - (VarExpr eI__P6)))) ")")
(FnCmd X7_ (((ArrayLValue k5V_ m4K OB__P) (BoolType) (ArrayLValue D K4_n Q63_) (IntType) (ArrayLValue r_ OW6s_ yS1_L) (ArrayType (StructType f_) 3))) (IntType) (AssertStmt (UnopExpr - (SumLoopExpr U_1I (VarExpr z) B0F_5 (IntExpr 965) u7iJ_3 (BinopExpr (IntExpr 4) - (VarExpr T)) h (BinopExpr (IntExpr 65542) % (VarExpr x)) (StructLiteralExpr Y__h1 (IfExpr (VarExpr t) (VarExpr e) (VarExpr j))))) "/v_,&^+5_") (LetStmt (VarLValue O_) (BinopExpr (BinopExpr (StructLiteralExpr kR (IntExpr 8586212278) (ArrayLoopExpr (VarExpr z)) (SumLoopExpr J8a_v (SumLoopExpr (VarExpr K)) M_T (VarExpr g) wike2 (VarExpr S) v (ArrayLoopExpr (VarExpr o)) (ArrayLoopExpr (BinopExpr (VarExpr F) + (VarExpr y)))) (CallExpr Aj4s__ (ArrayLoopExpr (VarExpr z))) (UnopExpr ! (VarExpr M))) != (ArrayLiteralExpr (CallExpr xi_ (VarExpr X)) (ArrayLoopExpr (DotExpr (VarExpr O) n)))) >= (StructLiteralExpr I_4G8_ (SumLoopExpr (VarExpr k))))) (ReturnStmt (StructLiteralExpr q_rx9 (CallExpr fh_US (IntExpr 580513705) (DotExpr (VarExpr t) s) (IntExpr 0)) (ArrayLiteralExpr (BinopExpr (VarExpr t) + (ArrayLoopExpr (VarExpr B))) (SumLoopExpr L (VarExpr K) (VarExpr G)) (IfExpr (DotExpr (VarExpr I) G) (IntExpr 721894138) (VarExpr m)) (ArrayLiteralExpr (VarExpr I))) (CallExpr V254__ (UnopExpr ! (VarExpr J)) (UnopExpr ! (VarExpr c))))) (AssertStmt (SumLoopExpr V418_ (IfExpr (VarExpr l_8__8) (IntExpr 5404) (IfExpr (CallExpr Z5z (VarExpr c)) (IfExpr (VarExpr L) (IntExpr 610579781) (VarExpr D)) (IntExpr 5523))) j_ (ArrayLoopExpr k (VarExpr T) F__f (SumLoopExpr (VarExpr E)) Zq0 (IntExpr 16067912) z7 (VarExpr L) (SumLoopExpr (ArrayLoopExpr (DotExpr (VarExpr z) y)))) X (ArrayLoopExpr (DotExpr (ArrayIndexExpr (DotExpr (VarExpr a) f) (IntExpr 220415908)) Y97S_n)) S2 (SumLoopExpr C (VarExpr mK98) i_4__ (ArrayLiteralExpr (VarExpr R)) D_3 (UnopExpr ! (VarExpr a)) b (BinopExpr (IntExpr 66) >= (VarExpr S)) y12_91 (IntExpr 8470082478) (ArrayLiteralExpr (VarExpr m) (IntExpr 28875) (IntExpr 8320638))) (CallExpr T___p (StructLiteralExpr j___2 (SumLoopExpr (VarExpr Y))))) "") (ReturnStmt (BinopExpr (DotExpr (StructLiteralExpr P_ (UnopExpr - (VarExpr c)) (IfExpr (VarExpr m) (VarExpr h) (VarExpr n)) (UnopExpr ! (VarExpr k)) (DotExpr (VarExpr S) R)) K1) && (ArrayLoopExpr wZ (StructLiteralExpr k8 (VarExpr h) (VarExpr Q) (VarExpr E) (DotExpr (VarExpr m) o)) (BinopExpr (ArrayLiteralExpr (VarExpr t) (IntExpr 21355) (VarExpr X) (IntExpr 43)) && (CallExpr R__ (IntExpr 39) (DotExpr (VarExpr x) M) (VarExpr N)))))))
